modules/dnn/test/test_onnx_importer.cpp

```diff
@@ -2164,7 +2164,7 @@ TEST_P(Test_ONNX_nets, Alexnet)
     expectNoFallbacksFromIE(net);
 }
 
-TEST_P(Test_ONNX_nets, RAFT)
+TEST_P(Test_ONNX_nets, DISABLED_RAFT)
```

Comment:
Please create ticket about the issue and add reference to it in comments.

Reply:
Reminder.

Reply:
https://github.com/opencv/opencv/issues/25102